charts/backstage/values.yaml

```diff
@@ -40,6 +48,50 @@ upstream:
             key: postgres-password
             name: "{{ .Release.Name }}-postgresql"
 
+    extraVolumeMounts:
```

Comment:
Can you please explain/document/comment on these inline?

Reply:
Done